src/assets/styles/base.scss

```diff
@@ -74,6 +74,10 @@ h3 {
   line-height: 2.25rem;
 }
 
+label {
+  font-size: 1rem;
```

Comment:
Will this affect other labels ? How are you verifying that?

Reply:
Hopefully all of them 😁 
This UI was a little bit behind on our modern designs, so this should bring them into sync with what you would see in the Edit ui for example. 

I just verified visually and it updated everything i'd expect it too. Certainly a little more global and perhaps out of scope of this ticket, but to meet requirements on my specific ticket i figured it made sense to update them all now.